src/App.js

```diff
@@ -1,13 +1,58 @@
 import React from 'react';
+import todo from './components/TodoComponents/Todo';
+import TodoForm from './components/TodoComponents/TodoForm'
+
+
+// const Task = props => {
+//   const { task } = props.taskProp;
+//   return (
+//     <div>
+//       <h5> {task} </h5>
+//     </div>
+//   );
+// };
+
 
 class App extends React.Component {
-  // you will need a place to store your state in this component.
-  // design `App` to be the parent component of your application.
-  // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor() {
+    super();
+    this.state = {
+      message: "Todo List: MVP",
+      tasks: todo,
+      updatedTasks: " "
+    };
+  }
+
+  handleUpdateState = () => {
+    const tasks = this.state.tasks.slice();
+    // //this clones our tasks array
+    // tasks.push ({
+    //   updatedTasks: "" //i want to push our new task into the tasks array
+    // });
+
+    this.setState({ tasks: [...this.state.tasks, {task:this.state.updatedTasks, id:Date.now(), completed:false }] });
```

Comment:
In order to clear your input field when a new Todo has been submitted
```suggestion
    this.setState({ tasks: [...this.state.tasks, {task:this.state.updatedTasks, id:Date.now(), completed:false }], updatedTasks: '' });
```